### DISCUSSION TITLE
Objet : Assistance urgente pour la connexion du compte Super Admin

### DISCUSSION THREAD

**Discussion by MonsieurMeless** [supabase/supabase#30763] [Questions]
Bonjour,

Lors de la conception de notre application avec votre service Lovable, nous rencontrons un problème persistant lors de la connexion au compte Super Admin. Une erreur (voir image ci-jointe) s'affiche à chaque tentative.

Nous tenons vraiment à finaliser ce projet avec votre service et sollicitons votre aide pour résoudre ce problème crucial, car la connexion est une fonctionnalité de base essentielle. Si ce problème persiste, cela pourrait gravement compromettre l’aboutissement de notre projet.

Nous comptons sincèrement sur votre expertise et espérons que vous pourrez nous apporter une solution rapidement. Merci de nous faciliter les choses en nous guidant dans ce sens.

Dans l’attente de votre retour avec impatience, nous vous remercions d’avance et vous souhaitons une excellente journée.

Bien cordialement,

**Comment by GaryAustin1** [comment]
You might get more results with English.

You mention you are seeking help with "your Loveable service" if the translation is correct.   Loveable is not a Supabase service but does have a Supabase integration I assume developed by Loveable.  You also say attached image has an error but there is no image.

**Reply by GaryAustin1** [reply]
You will have to give more info and you say there is an error in an image you don't link.  It is hard to know if Supabase related or not.

What is Super Admin account?  Not sure if that is a Loveable thing or something to do with Supabase's service_role key.
Ideally you would need to say what the Supabase call is that is getting the error and then possibly your table structure and RLS on that.  Really hard to know with no info.

**Comment by MonsieurMeless** [comment]
I understand, but could you please help me solve this problem? Since
Lovable uses Supabase and I am a professional developer, I am stuck at this
stage. If you manage to help me, I would be really happy, and we would be
extremely grateful to you.

Le dim. 1 déc. 2024 à 15:26, GaryAustin1 ***@***.***> a
écrit :
